doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -430,6 +430,8 @@ Other samples
 
     * Added new configuration that builds the sample with support for remote IPC Service shell on nRF5340 application core through USB.
     * Added possibility to build with the limited nRF21540 front-end module hardware pinout.
+    * Improved the calculation of the total payload size for the radio duty cycle.
+      In addition, fast ramp-up is enabled for all radio modes and duty cycle for modulated transmission is limited from 1 to 90%.
```

Comment:
Could we have these as separate bullets after the payload calculation item, something like:
* Fast ramp-up is enabled for all radio modes.
* The duty cycle for modulated transmission is limited to 1-90%.

Reply:
I think it's a good idea.